catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -68,10 +77,31 @@
 
     private boolean enableDelete = true;
 
+    private boolean initialized = false;
+
+    private ScheduledExecutorService executor;
+
+    private int taskWaitTimeSeconds = 30;
```

Comment:
`taskWaitTimeSeconds` is used in `refreshRegistryEntries`, which will run in the Camel thread if I'm not mistaken, and also in `setTimeWaitSeconds`, which is called by the Configuration Admin thread.

I should have taken a closer look the first time around. Looks like `refreshInterval` and `scheduledTask` are only used by a single thread, so those are ok the way they are.